src/ssz/type/bool.zig

```diff
@@ -14,6 +16,10 @@ pub fn BoolType() type {
             return a.* == b.*;
         }
 
+        pub fn clone(_: std.mem.Allocator, value: *const Type, out: *Type) !void {
```

Comment:
```suggestion
        pub fn clone(value: *const Type, out: *Type) !void {
```

Reply:
Addressed in [88f2fb0](https://github.com/ChainSafe/ssz-z/pull/51/commits/88f2fb0366abfeaaeb296bc736982d700f89feed)